### DISCUSSION TITLE
When I restart a vscode, tailwind css is broken

### DISCUSSION THREAD

**Discussion by lidarbtc** [tailwindlabs/tailwindcss#8942] [Help] [answered]
Yes when I restart a vscode, css dosen't work.

But If I type
```
npm run dev
```
on terminal again, css work again.

What should I do?

my os : ubuntu 22.04
ide : vscode

**Comment by wongjn** [comment, marked as answer]
Could you give more information about how the CSS doesn't work? Or more details about your project? It could be that the CSS is kept only in memory when you run `npm run dev` – this is how some build tools handle development mode, for example Webpack dev server.

**Reply by lidarbtc** [reply]
I don't know but it work again lol. have a good day. 👍

**Reply by Bravim-Ketan-Purohit** [reply]
@wongjn how do we use the tools like webpack dev server
